bridge/src/main/java/io/stargate/bridge/service/BridgeService.java

```diff
@@ -93,9 +93,17 @@ public void executeQueryWithSchema(
       StreamObserver<Schema.QueryWithSchemaResponse> responseObserver) {
     String keyspaceName = request.getKeyspaceName();
     int keyspaceHash = request.getKeyspaceHash();
-    String decoratedName =
-        persistence.decorateKeyspaceName(keyspaceName, BridgeService.HEADERS_KEY.get());
-    Keyspace keyspace = persistence.schema().keyspace(decoratedName);
+    String decoratedName;
+    Keyspace keyspace;
+
+    try {
+      decoratedName =
+          persistence.decorateKeyspaceName(keyspaceName, BridgeService.HEADERS_KEY.get());
+      keyspace = persistence.schema().keyspace(decoratedName);
+    } catch (Throwable t) {
+      new ExceptionHandler(responseObserver).handleException(t);
+      return;
+    }
```

Comment:
I am gonna include the whole method with try.catch, no need just to protect that first part.

Reply:
Most likely also needs rebase after #2190 is merged